extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -400,45 +704,97 @@ A dialog is marked up using the `subscriptions-dialog` and `subscriptions-displa
 The element on which `subscriptions-dialog` dialog is specified can also be a `<template>` element in which case it will be initially rendered before being displayed as a dialog. For instance:
 
 ```html
-<template type="amp-mustache" subscriptions-dialog subscriptions-display="EXPR">
-  <div>
-    You have {{metering.left}} articles left this month.
-  </div>
+<template type="amp-mustache" subscriptions-dialog subscriptions-display="NOT granted">
+  <!-- Customise the experience for the user using the `data` object returned in the authorization response -->
+  <!-- Do NOT use the `data` object to show or hide premium content as this is not always returned -->
+  {{^data.articlesRead}}
+  <p>
+    You have read all of your free articles!
+  </p>
+  {{/data.articlesRead}}
+  {{#data.articlesRead}}
+  <p>
+    You have read <b>{{data.articlesRead}}</b> articles.
+  </p>
+  {{/data.articlesRead}}
+  {{#data.articlesLeft}}
+  <p>
+    You have <b>{{data.articlesLeft}}</b> free articles left!
+  </p>
+  {{/data.articlesLeft}}
+  <button subscriptions-action="subscribe" subscriptions-service="local" subscriptions-display="true">
+    Subscribe
+  </button>
+  <section subscriptions-display="NOT granted AND NOT data.isLoggedIn">
+  <button
+    subscriptions-action="login"
+    subscriptions-service="local"
+    subscriptions-display="NOT granted AND NOT data.isLoggedIn">
+    Already subscribed?
+  </button>
+
 </template>
 ```
 
 The first dialog with matching `subscriptions-display` is shown.
 
-## Expressions
+## Actions
```

Comment:
Could you provide an example of how these actions would be called? I'm unsure of the syntax, but the flow information is great

Reply:
Done - thanks for all your detailed reviews and comments!